dockerfiles/Dockerfile.nemo-skills

```diff
@@ -1,12 +1,26 @@
-FROM python:3.10-bookworm
-
-RUN apt-get update && apt-get -y install curl git git-lfs
-
-# installing apptainer
-RUN apt install -y wget && \
-    cd /tmp && \
-    wget https://github.com/apptainer/apptainer/releases/download/v1.4.1/apptainer_1.4.1_amd64.deb && \
-    apt install -y ./apptainer_1.4.1_amd64.deb
+# using ubuntu instead of debian for easier apptainer installation on arm64
+FROM ubuntu:22.04
+
+# Install Python and other dependencies
+RUN apt-get update && \
+    apt-get install -y \
+    python3.10 \
+    python3-pip \
+    curl \
+    git \
+    git-lfs && \
+    ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
Always a good practice to use:
```shell
rm -rf /var/cache/apt/archives /var/lib/apt/lists/*
```
in every layer that uses `apt`. Significantly reduces image size.

See https://docs.docker.com/build/building/best-practices/#apt-get